packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -422,13 +422,15 @@ function ListView({
 
     if (reviewWorkflowColumns) {
       // Make sure the column header label is translated
-      if (typeof reviewWorkflowColumns.metadatas.label !== 'string') {
-        reviewWorkflowColumns.metadatas.label = formatMessage(
-          reviewWorkflowColumns.metadatas.label
-        );
-      }
+      reviewWorkflowColumns.map((column) => {
```

Comment:
What is this in CE? I'm worried it might be `null`.

```suggestion
      (reviewWorkflowColumns ?? []).map((column) => {
```